src/person/tasks.py

```diff
@@ -35,8 +37,24 @@ def update_paused_users():
     every=12,
     period=IntervalSchedule.HOURS,
 )
+
 PeriodicTask.objects.get_or_create(
     interval=schedule,
     name="Pause Pear Updater",
     task="person.tasks.update_paused_users",
 )
+
+
+@shared_task
+def update_inactive_users():
+    three_weeks_ago = datetime.now() - timedelta(days=21)
+    inactive_users = Person.objects.filter(last_active__lt=three_weeks_ago)
+    inactive_users.update(is_paused=True, pause_expiration=None)
+    return f"Unpaused {len(inactive_users)} inactive users"
```

Comment:
what is the point of returning out of a task function? are you expecting to see this return statement printed somewhere?

Reply:
it may actually spit this out in the celery logs, have you verified this? @jzs27